.gitlab-ci.yml

```diff
@@ -22,7 +22,7 @@ variables:
 
 .kas-build:
   stage: build
-  image: ghcr.io/siemens/kas/kas-isar:3.0.2
+  image: ghcr.io/siemens/kas/kas-isar:3.1
```

Comment:
What about the kas-container copy in this layer?

Reply:
Good catch. Fixed.

Reply:
Then maybe also rename the subject as you are no longer only targeting "ci:".